### PR TITLE
doc: add entrypoint and command to manifest documentations

### PR DESCRIPTION
**Pull request by Lou1415926**
Adds docs on `entrypoint` and `command` fields in manifest files.

Preceding PR: #1999 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: site/content/docs/manifest/backend-service.md [hunk 1]
```diff
@@ -121,6 +121,31 @@ Grace period within which to provide containers time to bootstrap before failed
...
 
+<a id="entrypoint" href="#entrypoint" class="field">`entrypoint`</a> <span class="type">String or Array of Strings</span>  
+Override the default entrypoint in the image. 
+```yaml
+entrypoint: /bin/entrypoint --p1 --p2
+```
+
+```yaml
+entrypoint: ["/bin/entrypoint", "--p1", "--p2"]
+```
```
Comment (efekarakus) [inline site/content/docs/manifest/backend-service.md]: Just to conserve some real-estate:
```suggestion
```yaml
# String version.
entrypoint: "/bin/entrypoint --p1 --p2"
# Alteratively, as an array of strings.
entrypoint: ["/bin/entrypoint", "--p1", "--p2"]
```
```

File: site/content/docs/manifest/backend-service.md [hunk 2]
```diff
@@ -121,6 +121,29 @@ Grace period within which to provide containers time to bootstrap before failed
...
 <div class="separator"></div>
 
+<a id="entrypoint" href="#entrypoint" class="field">`entrypoint`</a> <span class="type">String or Array of Strings</span>  
+Override the default entrypoint in the image. 
+```yaml
+# String version.
+entrypoint: "/bin/entrypoint --p1 --p2"
+# Alteratively, as an array of strings.
+entrypoint: ["/bin/entrypoint", "--p1", "--p2"]
+
```
Comment (huanjani) [inline site/content/docs/manifest/backend-service.md:132]: I think we're missing the closing ``` here.

File: site/content/docs/manifest/backend-service.md [hunk 3]
```diff
@@ -121,6 +121,29 @@ Grace period within which to provide containers time to bootstrap before failed
...
+<div class="separator"></div>
+
+<a id="command" href="#command" class="field">`command`</a> <span class="type">String or Array of Strings</span>  
+Override the default command in the image.
+
```
Comment (huanjani) [inline site/content/docs/manifest/backend-service.md:137]: nit: Should we have the same comments as above? It's redundant, but this is sort of a menu, so people may only read the entries they're looking into. (here and below)

Comment (Lou1415926) [reply site/content/docs/manifest/backend-service.md:137]: Yess fixed now! I committed efe's suggestions directly from here and forgot to fix the rest ughhh

**Review by huanjani — APPROVED** [review]
LGTM!

**Review by efekarakus — APPROVED** [review]

**Pull request merged**